astrodendro/tests/test_io.py

```diff
@@ -59,93 +51,99 @@ def compare_dendrograms(self, d1, d2):
 
     # Below are the actual tests for each import/export format:
 
-    def test_hdf5(self):
-        self.test_filename = 'astrodendro-test.hdf5'
+    def test_hdf5(self, tmpdir):
+        test_filename = tmpdir.join('astrodendro-test.hdf5').strpath
```

Comment:
just want to verify... `tmpdir.join(str)` will return `tmpdir/str` ?  Naively I would expect `join` to treat its input as an iterable and try to join (e.g., `os.path.join(x,y)` returns `f'{x}/{y}'`)